### ISSUE TITLE
'async' is a reserved word in Python 3.7

### ISSUE THREAD

**Issue by cclauss** [diux-dev/imagenet18#2]
__async__ is a reserved word in Python 3.7 so it is a Syntax Error to use it as the name of a function parameter.  Some projects like __cuda__ are using __non_blocking__ instead of __async__.

[flake8](http://flake8.pycqa.org) testing of https://github.com/diux-dev/imagenet18 on Python 3.7.0

$ __flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics__
```
./train.py:192:23: E999 SyntaxError: invalid syntax
    task.run(cmd, async=True)
                      ^
./tools/launch_tensorboard.py:14:56: E999 SyntaxError: invalid syntax
task.run(f'tensorboard --logdir={task.logdir}/..', async=True)
                                                       ^
2     E999 SyntaxError: invalid syntax
2
```

**Comment by yaroslavvb** [comment]
Oh wow, this is a crash and not just a warning in 3.7? I thought backward compatibility breaking was reserved for major version number change

**Comment by cclauss** [comment]
Not really possible in a language that _only has three Major versions_ while v1.0 was released in 1994.  
https://devguide.python.org/devcycle

**Comment by yaroslavvb** [comment]
Fixed by #6
